config/foreman-answers.yaml

```diff
@@ -11,6 +11,17 @@
 ---
 foreman:
   configure_epel_repo: false
+  server_ssl_crl: ""
+  server_ssl_ca: "/etc/foreman-pki/certs/ca/ca.crt"
+  server_ssl_chain: "/etc/foreman-pki/certs/ca/ca.crt"
```

Comment:
I don't like this scheme naming. It implies a single CA but we know that's not true. I'd suggest at least symlinks: `/etc/foreman-pki/certs/apache/apache-ca.pem`

Reply:
There is still "custom certificates" to come support wise which is why this is currently this way. It is built around a single CA concept as the intro use case. I'm still contemplating how to handle custom certificates and at what level. I think it would be nice if we could get away with the simple case of supplying a single CA rather than customizing only some certificates but feedback so far is that is not feasible for all use cases.

Reply:
If we already use bundles of 3 (cert, key, ca) then we can easily support that in the future. If you symlink to the common CA, you still only have a single file. Perhaps we should also have a 4th (chain) for when we have client auth.

I'd also recommend looking at certbot (the Let's Encrypt client). It creates a structure:
* cert.pem (certificate)
* chain.pem (the CA chain, including intermediates)
* fullchain.pem (the full chain - from CA to cert)
* privkey.pem (private key)

It does so for every certificate. Perhaps we should follow the same where the directory indicates the purpose and the file names within the directory are always the same. That way you could create a bundle `private-ca` and a bundle `public-ca`. Then just symlink `apache` to `public-ca` and you're done. However, perhaps file permissions will get in the way.